draw.c

```diff
@@ -150,6 +150,11 @@ void draw_splits(struct state *s, int w, int h, int *y, int off, struct split *s
 			cairo_fill(s->cr);
 		}
 
+		// For the active split, we want to display the delta as soon as it goes over gold
+		if (active && (s->split_time > times.best)) {
```

Comment:
What should the behaviour be here if there's no gold currently associated with the split? This will currently just never show an active delta in that case; is this what we want?

Reply:
I think this is fine and it is what livesplit does, if I'm not mistaken.

Reply:
Yep, okay - I wasn't saying it was wrong, just checking that it made sense